### ISSUE TITLE
Display error messages in OSD for some seconds

### ISSUE THREAD

**Issue by Nojevah** [mpv-player/mpv#7631]
### Expected behavior of the wanted feature
I'd like to have errors messages from terminal displayed in OSD, since in general I don't launch file with mpv.com

![image](https://user-images.githubusercontent.com/7541249/79635852-c1bf8e00-8173-11ea-8be6-50ab9c2e2dde.png)

For example, I'd like these red messages to be displayed on OSD for some seconds.

I though it was possible, but re-reading the manual, I found nothing.

**Comment by CounterPillow** [comment]
This is a pretty good idea in that it could integrate with console.lua in some way. E.g. have the console show up in a non-input-grabbing way temporarily.

Should probably also have an option to disable this from happening though, in case someone has a setup where they experience some error message that is non-fatal to their use case and don't want to be told about it while watching a video.

**Comment by wm4** [comment]
Yeah, the console (which is now enabled by default), but still needs manual interaction to show it. Showing errors without interaction might be useful, but we don't know which errors are actually interesting enough. Especially FFmpeg tends to log "errors" that, depending on the context, can be ignored. We don't control these, and in general we can't really tell.

**Comment by CogentRedTester** [comment]
While having the console pop up would undoubtedly be better, it doesn't seem too difficult to write a script to have error messages print to the osd. I wrote a quick script to do so, but there may be a few bugs:

```
local ov = mp.create_osd_overlay("ass-events")

mp.enable_messages('error')
mp.register_event('log-message', function(log)
    message = "{\\c&H0000CC>&}[" .. log.prefix .. "] " .. log.text
    ov.data = ov.data .. message
    ov:update()

    mp.add_timeout(4, function ()
        local endln = ov.data:find('\n') + 1
        ov.data = ov.data:sub(endln)
        ov:update()
    end)
end)
```

**Comment by ZhenFTW** [comment]
I am looking for something similar to this and found nothing. I thought there was a way to do it, since ffmpeg clearly shows the error in the console by printing the error in red color or yellow in warning. I wonder why there is less appeal to this issue. It is important feature for the mpv api to detect the issue and get the string of an error. Hope there's an event for video playback error handling in mpv like vlc and exoplayer.
